### ISSUE TITLE
Not on the Chrome Extension Store?

### ISSUE THREAD

**Issue by LocutusX** [bfred-it/sticky-pinned-tabs#2]
I'd like to try this extension, but it's not on the Chrome extension store. Can you please re-add it there?

**Comment by fregante** [comment]
I don’t think it was working anymore, or maybe it just had too many bugs, so I removed it
